lib/matplotlib/table.py

```diff
@@ -314,18 +335,21 @@ def __init__(self, ax, loc=None, bbox=None, **kwargs):
 
     def add_cell(self, row, col, *args, **kwargs):
         """
-        Add a cell to the table.
+        Create a cell and add it to the table.
 
         Parameters
         ----------
         row : int
             Row index.
         col : int
             Column index.
+        *args, **kwargs
+            All other parameters are passed on to `Cell`.
 
         Returns
         -------
-        `CustomCell`: Automatically created cell
+        cell : `CustomCell`
```

Comment:
still no backquotes needed I think?

Reply:
No, the backquotes are intended to link to `CustomCell`:
![image](https://user-images.githubusercontent.com/2836374/49607213-9b8db980-f995-11e8-9fff-a3ff662d84e4.png)

I've added a leading .